src/relayserver/runServer.ts

```diff
@@ -68,6 +69,7 @@ const ethereumNodeUrl: string = argv.ethereumNodeUrl ?? config.ethereumNodeUrl ?
 const workdir: string = argv.workdir ?? config.workdir ?? error('missing --workdir')
 const devMode: boolean = argv.devMode ?? config.devMode ?? error('missing --devMode')
 const debug: boolean = argv.debug ?? config.debug ?? error('missing --debug')
+const registrationBlockRate: string = argv.registrationBlockRate ?? config.registrationBlockRate?.toString()
```

Comment:
`null?.toString()` will return 'undefined' and this should break the types system here...

Reply:
Wouldn't that crash?

Reply:
And it will!..